plugins/modules/b1_ipam_fixed_address.py

```diff
@@ -432,6 +436,11 @@ def main():
     }
 
     module = AnsibleModule(argument_spec=argument_spec)
+    module.deprecate(
+        "This module is being deprecated and will be removed in version 3.0.0. Use the M(ipam_fixed_address) module instead.",
+        version="3.0.0",
+        collection_name="infoblox.bloxone",
+    )
```

Comment:
Why was this needed? 

I am asking, since this is not mentioned in https://docs.ansible.com/ansible/latest/dev_guide/module_lifecycle.html#deprecating-modules
Also, we haven't done this for the other deprecated modules. Will like to be consistent if possible.